### PR TITLE
edit-all-folders-and-files

### PR DESCRIPTION
**Pull request by amirhosein0980**
The names of all folders and their files were edited
Please check the files to merge

### CODE REVIEW

File: src/services/presentationservice.ts
Comment (chalist) [inline src/services/presentationservice.ts:1]: return this change! first one was correct. `presentation.service.ts`

Comment (amirhosein0980) [reply src/services/presentationservice.ts:1]: Edited

File: src/pages/Authentication/Sign-up/Sign-up.jsx
[the remark this answers is not in the document]
Comment (amirhosein0980) [reply src/pages/Authentication/Sign-up/Sign-up.jsx:1]: Edited

File: src/pages/Authentication/Sign-in/Sign-in.jsx
[the remark this answers is not in the document]
Comment (amirhosein0980) [reply src/pages/Authentication/Sign-in/Sign-in.jsx:1]: Edited

File: src/pages/Authentication/Reset-password/Reset-password.jsx
[the remark this answers is not in the document]
Comment (amirhosein0980) [reply src/pages/Authentication/Reset-password/Reset-password.jsx:1]: Edited

**Pull request closed without merging**